appveyor.yml

```diff
@@ -46,6 +40,9 @@ platform:
     - x64
 
 install:
+    # Skip if we are on `master`.
+    - cmd: if call git rev-parse --abbrev-ref HEAD == "master" exit 0
```

Comment:
Try this instead `git rev-parse --abbrev-ref HEAD | findstr /v "master"`. Errors if there is a match.